cmds/rm/rm.go

```diff
@@ -69,10 +64,10 @@ func rm(files []string) error {
 			return err
```

Comment:
What does `rm` on your machine do when you try to remove a file that doesn't exist? If you give it more than one file, and the first one doesn't exist, does it try to delete the others?